content/docs/rfcs/31/README.md

```diff
@@ -118,8 +118,42 @@ Supported key type is `secp256k1` only.
 
 In the future, an extension of this RFC could be made to support other elliptic curve cryptography such as `ed25519`.
 
+# `waku2` ENR key
+
+We propose the definition of a `waku2` field key:
+
+- The value MUST be an 8-bit flag field,
+where bits set to `1` indicate `true` and bits set to `0` indicate `false` for the relevant flags.
+- The flag values already defined are set out below,
+with `bit 7` the most significant bit and `bit 0` the least significant bit.
+
+| bit 7 | bit 6 | bit 5 | bit 4 | bit 3 | bit 2 | bit 1 | bit 0 |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| `undef` | `undef` | `undef` | `undef` | `lightpush` | `filter` | `store` | `relay` |
+
+- In the scheme above, the flags `lightpush`, `filter`, `store` and `relay` correlates with support for protocols with the same name.
+If a protocol is not supported, the corresponding field MUST NOT be set to `true`.
```

Comment:
MUST be set to `false`? Or do you see a difference?

Reply:
Try as I might I can't get these to parse differently in my mind this morning :). MUST be `false` is indeed clearer. The point is, of course, that `false` does not necessarily mean that the client does not support that protocol, just that they choose not to make that fact discoverable.

Reply:
Done in 27bd3a8.